app/frontend/src/locales/fr/translation.json

```diff
@@ -61,7 +61,9 @@
         "showSupportingContent": "Montrer le contenu de soutien",
         "speakAnswer": "Parler réponse",
         "info": "Info",
-        "save": "Sauvegarder"
+        "save": "Sauvegarder",
+        "copy": "Copier la réponse",
+        "copied": "Copié!"
```

Comment:
With my basic knowledge of French, this looks correct to me.

Reply:
Wassim reviewed it, french is good to go!